website/app/templates/head.hbs

```diff
@@ -38,19 +38,16 @@ We've made some small customizations to simplify this compared to the default te
   <meta property="og:description" content={{this.model.description}} />
 {{/if}}
 
-{{#if this.model.imgSrc}}
-  <meta property="og:image" content={{this.model.imgSrc}} />
-{{/if}}
+<meta property="og:image" content="https://helios.hashicorp.design/assets/logos/share-card.jpg" />
```

Comment:
A side effect of this change is that we are no longer using the logic defined [here](https://github.com/hashicorp/design-system/blob/main/website/app/services/head-data.js#L20) to update the image where we have a more specific one defined for things like components or foundations

Reply:
Opened #1158 to take in account also this logic.

Reply:
@Dhaulagiri on a side note: that's why I didn't like the idea of having the config data in the `environment.js` file (even it's an Ember pattern). Because now things/logic/definitions are scattered around in multiple places, and one needs to know it, and know where, otherwise some pieces are lost (like in this case).